src/System.Net.WebHeaderCollection/ref/System.Net.WebHeaderCollection.cs

```diff
@@ -5,6 +5,7 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+#pragma warning disable CS8610
```

Comment:
I think we should do this in the ref .csproj instead; otherwise we're going to have to redo this every time we regenerate the refs.

Reply:
(If we want to do it in the source, then I think we should do it around every case of this rather than at the top of the file, so that we still get warnings for unexpected cases... right now with this at the top of the file, it feels like a middle ground with the worst of both worlds.)

Reply:
That makes sense, I think it would be better to have it in the ref .csproj instead, the reason why I didn't want to do it around every case it is because every time we generate this file, it will cause pain to whoever generates it. If the implementation is annotated correctly and suppresses only the warnings that it needs to suppress, it should technically generate correct ref assembly + when I add APICompat validation it should catch miss match of nullable annotations in between src and ref.